ledger/metrics/src/main/scala/com/daml/telemetry/SpanName.scala

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.telemetry
+
+object SpanName {
```

Comment:
You may just inline this as it is only being used from two independent modules.

Reply:
I'd like to gather all future values here, similarly to the oem integration kit.

Reply:
In that case you might want to call this as `SpanNames`.